features/zoek-met-geslachtsnaam-geboortedatum.feature

```diff
@@ -0,0 +1,67 @@
+#language: nl
+
+Functionaliteit: Zoek met geslachtsnaam en geboortedatum
+
+Rule: Geslachtsnaam en geboortedatum zijn verplichte parameters
+
+    Scenario: Zoek met valide geslachtsnaam en geboortedatum
+
+        Gegeven het systeem heeft een persoon met de volgende gegevens
+        | naam                | waarde    |
+        | burgerservicenummer | 999995078 |
+        En de persoon heeft de volgende geboorte gegevens
+        | naam        | waarde   |
+        | datum.type  | GbaDatum |
+        | datum.datum | 19830526 |
+        En de persoon heeft de volgende naam gegevens
+        | naam                  | waarde  |
+        | geslachtsnaam (02.40) | Maassen |
+        Als personen wordt gezocht met de volgende parameters
+        | naam          | waarde                              |
+        | type          | ZoekMetGeslachtsnaamEnGeboortedatum |
+        | geslachtsnaam | Maassen                             |
+        | geboortedatum | 1983-05-26                          |
+        | fields        | burgerservicenummer                 |
+        Dan bevat de response alleen personen met de volgende gegevens 
+        | burgerservicenummer |
+        | 999995078           |
+        En bevat de persoon met burgerservicenummer '999995078' de volgende geboorte gegevens
```

Comment:
fields=burgerservicenummer, dus geboorte en naam krijg je nu niet

Reply:
> fields=burgerservicenummer, dus geboorte en naam krijg je nu niet

Dit begrijp ik niet, want ik beschrijf met deze scenario precies één functionaliteit namelijk dat een persoon met geslachtsnaam X en geboortedatum Y ook gevonden wordt als X en Y als zoek parameter waarde wordt opgegeven. Deze scenario beschrijft niet dat als een persoon een geslachtsnaam heeft en ik geef fields=geslachtsnaam op als parameter dat de response geslachtsnaam bevat.
Als ik dit zou doen, maak ik deze scenario broos. Een bug in de fields parameter afhandeling implementatie zou kunnen leiden dat deze scenario ook faalt.

Reply:
In het scenario zeg je "Als personen wordt gezocht met de volgende parameters" ... | fields | burgerservicenummer |
En bij de Dan zeg je "En bevat de persoon met burgerservicenummer '999995078' de volgende geboorte gegevens"

Wanneer je een request doet met fields=burgerservicenummer, dan krijg je alleen burgerservicenummers terug. Het gebruik van parameters heeft geen invloed op welke velden je terugkrijgt.
Dus om het scenario kloppend te krijgen, moet je fields uitbreiden naar fields=burgerservicenummer,geboorte.datum,naam.geslachtsnaam

Reply:
my mistake, toegevoegd